packages/es-components/src/components/containers/tabPanels/Tab.js

```diff
@@ -44,7 +45,7 @@ const TabButton = styled.button`
   }
 `;
 
-const AriaAnnouncer = styled.span`
+const AriaAnnouncer = styled.p`
   position: fixed;
```

Comment:
I've seen a couple of different methods used lately in our code to hide screenreader content, and maybe we should pick one and stick to it. [WebAIM](https://webaim.org/techniques/css/invisiblecontent/) has a good article on this and has a good (similar) solution. See the "Positioning Content Off-screen" section of that article. They add a few more rules to make sure there aren't any visual artifacts or other issues with this method.

Reply:
I can add those extra properties